src/Pages/Main/Food/FoodContent.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+class FoodContent extends React.Component {
+  componentDidMount() {}
+
+  render() {
+    const labelColor = {
+      냉장: '#686de0',
+      냉동: '#7ed6df',
+      실온: '#f0932b',
+      싱싱: '#badc58',
+    };
+    const { ingredients } = this.props;
+    const labelName = ingredients.storage;
+    return (
+      <div className="food">
```

Comment:
- 컴포넌트 이름과 동일하게 지어주세요!
```suggestion
      <div className="foodContent">
```

Reply:
수정했습니다!